### PR TITLE
live: remove summary

### PR DESCRIPTION
**Pull request by daavoo**
P.R. for https://github.com/iterative/dvclive/pull/217

### CODE REVIEW

**Review by daavoo — COMMENTED** [review]
@dberenbaum not sure what to do with https://github.com/iterative/dvclive/issues/211#issuecomment-1036638157

> In the meantime, we can document that `summary: false`/`--live-no-summary` will be ignored in `dvclive>=0.6.0`.

**Review by shcheklein — APPROVED** [review]

**Comment by dberenbaum** [comment]
> @dberenbaum not sure what to do with [iterative/dvclive#211 (comment)](https://github.com/iterative/dvclive/issues/211#issuecomment-1036638157)



> @dberenbaum not sure what to do with [iterative/dvclive#211 (comment)](https://github.com/iterative/dvclive/issues/211#issuecomment-1036638157)

Right, I guess it doesn't make much sense to mention the summary if it's now being ignored. Probably the warning on the dvclive side about deprecating that option was sufficient. This is where having versioned docs (at least for the API reference) would be nice.

@shcheklein Any thoughts on whether we should address that `--live-no-summary` will have no effect for `dvclive>=0.6`?

**Review by dberenbaum — APPROVED** [review]

**Comment by dberenbaum** [comment]
> @shcheklein Any thoughts on whether we should address that `--live-no-summary` will have no effect for `dvclive>=0.6`?

Ping @shcheklein 

@daavoo Do you want to release 0.6 before merging?

**Comment by shcheklein** [comment]
If understand correctly, I think the way we do this (just getting rid of this in docs) is totally fine, it's better to move faster here and not complicate too much.

File: content/docs/command-reference/run.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ usage: dvc run [-h] [-q | -v] [-n <name>] [--no-exec] [--no-run-cache]
                [--no-commit] [-d <path>] [-o <filename>] [-O <filename>]
                [-p [<filename>:]<params_list>] [-m <path>] [-M <path>]
                [--plots <path>] [--plots-no-cache <path>] [--live <path>]
-               [--live-no-cache <path>] [--live-no-summary] [--live-no-html]
+               [--live-no-cache <path>] [--live-no-html]
                [-w <path>] [-f] [--outs-persist <filename>]
                [--outs-persist-no-cache <filename>] [-c <filename>]
                [--always-changed] [--external] [--desc <text>]
```
Comment (jorgeorpinel) [inline content/docs/command-reference/run.md:16]: Minor: could probably rewrap this differently now. Maybe with one less line

File: content/docs/command-reference/stage/add.md [hunk 2]
```diff
@@ -9,8 +9,8 @@ usage: dvc stage add [-h] [-q | -v] -n <name> [-d <path>] [-o <filename>]
                      [-O <filename>] [-p [<filename>:]<params_list>]
                      [-m <path>] [-M <path>] [--plots <path>]
                      [--plots-no-cache <path>] [--live <path>]
-                     [--live-no-cache <path>] [--live-no-summary]
-                     [--live-no-html] [-w <path>] [-f]
+                     [--live-no-cache <path>] [--live-no-html]
+                     [-w <path>] [-f]
                      [--outs-persist <filename>]
                      [--outs-persist-no-cache <filename>] [-c <filename>]
                      [--always-changed] [--external] [--desc <text>]
```
Comment (jorgeorpinel) [inline content/docs/command-reference/stage/add.md:16]: Same

**Pull request merged**